repos/suse.template.repos

```diff
@@ -80,6 +80,6 @@ https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifac
 https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Backports/SLE-15-SP2_x86_64/standard?auth=basic                                  SUSE-Backports-SLE-15-SP2-x86_64                                      -g -p 89  suse/Backports-SLE/15-SP2/x86_64/standard
                                                                                                                                                                               
 # cephadm, ceph-common, and libfmt7                                                                                                                                               
-https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/opensuse-mirror/openSUSE:Backports:SLE-15-SP3/step/                                          openSUSE-Backports-SLE-15-SP3-step                                    -g -p 89  openSUSE:Backports:SLE-15-SP3/step/
-https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/opensuse-mirror/openSUSE:Backports:SLE-15-SP3/standard/                                      openSUSE-Backports-SLE-15-SP3-standard                                -g -p 89  openSUSE:Backports:SLE-15-SP3/standard/
-https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/opensuse-mirror/filesystems:ceph/openSUSE_Leap_15.3/                                         filesystems-ceph                                                      -g -p 89  openSUSE_Leap_15.3/
+https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/opensuse-mirror/openSUSE:Backports:SLE-15-SP4/step/                                          openSUSE-Backports-SLE-15-SP4-step                                    -g -p 89  openSUSE:Backports:SLE-15-SP4/step/
```

Comment:
We haven't moved to SP4 yet, I'm in a holding pattern with that right now. If ceph*17.2.3 needs packages from SP4 repos we can hold this until then.